tests/integration/test_config_hidden_attributes/configs/config.xml

```diff
@@ -0,0 +1,11 @@
+<clickhouse>
+    <max_table_size_to_drop hidden="true">60000000000</max_table_size_to_drop>
```

Comment:
Thinking about naming. "hidden" as attribute name is not as intuitive as existing attributes "from_env" or "from_zk" or "encrypted_by". What about "hide_in_preprocessed" instead?

Reply:
Done. Renamed.